src/webhook_dispatcher.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+def post_to_webhook(webhook_url, job_id, filename, url, success=False):
+    payload = {
+        "source": "waas",
+        "job_id": job_id,
+        "success": success
+    }
+    if success:
+        payload["url"] = url
+        payload["filename"] = filename
+    try:
+        req = requests.post(webhook_url, json=payload, allow_redirects=False, timeout=3)
+        req.raise_for_status()
```

Comment:
Neat, I didn't know about this!

Reply:
We might need to add some retry logic in the future, but at least this won't block for more than 3 seconds and probably not end up in a redirect loop.